docs/best-practices/migrating-from-mysql.rst

```diff
@@ -74,13 +74,12 @@ when read from the convert script later.
 Converting date/time types
 --------------------------
 
-The standard output for `date/time types in MySQL
-<http://dev.mysql.com/doc/refman/5.0/en/date-and-time-types.html>`_ is a
-string. However, CrateDB uses a ``long`` type to store timestamps (with
-millisecond precision). To prevent problems with dates that have timezone
-information, use MySQL's builtin ``UNIX_TIMESTAMP`` function to convert
-``date``, ``time``, ``datetime``, ``timestamp`` and ``year`` types into UNIX
-timestamps directly in the ``SELECT`` statement.
+The standard output for `date/time types in MySQL`_ is a string. However,
+CrateDB uses a ``long`` type to store timestamps (with millisecond precision).
+To prevent problems with dates that have timezone information, use MySQL's
+builtin ``UNIX_TIMESTAMP`` function to convert ``date``, ``time``,
+``datetime``, ``timestamp`` and ``year`` types into UNIX timestamps directly in
```

Comment:
Last Oxford comma after 'timestamp'?

Reply:
good catch. thx! I'm gonna leave this in as it is unrelated to the changes and it's good to keep things separate (it makes backporting and cherry-picking a lot easier when changes are self-contained)